Doxyfile

```diff
@@ -1152,7 +1152,7 @@ COLS_IN_ALPHA_INDEX    = 5
 # while generating the index headers.
 # This tag requires that the tag ALPHABETICAL_INDEX is set to YES.
 
-IGNORE_PREFIX          =
+IGNORE_PREFIX          = az_
```

Comment:
This is case sensitive. Should be `az_` and `AZ_`

Reply:
Right, I was going to ask why `AZ_CREDENTIAL_ANONYMOUS` was showing up under -a- in the screenshot you shared.

Reply:
Here's what it looks like when we do that. I'm not sure if it's more helpful...

![image](https://user-images.githubusercontent.com/2158838/80634318-24841580-8a0f-11ea-8f99-e22684d9aa0b.png)